src/model/artboard.js

```diff
@@ -584,7 +586,7 @@ class Artboard {
                                                                                     }.bind(this)
                                                                                 );
                                                                         } else {
-                                                                            // filemanager.deleteFile(file.path);
+                                                                            filemanager.deleteFile(file.path);
```

Comment:
This should be safe. I see no probs with it, especially because `deleteFile` is being used in other places.